share/extender/build_input.yml

```diff
@@ -122,15 +122,15 @@ platforms:
 
   osx:
     env:
-        PATH:                   "{{env.PLATFORMSDK_DIR}}/XcodeDefault{{env.XCODE_15_VERSION}}.xctoolchain/usr/bin:{{env.PATH}}"
+        PATH:                   "{{env.PLATFORMSDK_DIR}}/XcodeDefault{{env.XCODE_16_VERSION}}.xctoolchain/usr/bin:{{env.PATH}}"
```

Comment:
Same comment as Evgeniy here.
Now that we have individual servers, we don't need to do this version maintenance all over.
If we move to `env.XCODE_VERSION` etc instead, any future update should be a lot easier.

Reply:
not for is. For ios we still use one server as I understand